citi-onboarding/Recorda-one-page
# feat(Feature/Feedbacks_CRUD): Finished CRUD for Feedbacks section

#What I did:
- Created the model for Feedbacks section
- Created the Controller
- Created the routes for the FeedbackController
- Deleted files related to User CRUD 
...

#How to test
- git checkout main
- git pull
- git checkout feature/Feedbacks_CRUD
- git pull
- Open terminal 
- Make sure you haver /sever opened at the terminal
- Type "docker-compose up" then press Enter
- Open Insominia and Postbird
- Test with "get", "post", "put" and "delete" at Insominia
- Check out at Postbird if everything is working correctly
...
